Add O lettered article stubs from glossary (#3859)
ppy/osu-wiki: wiki/osu!coin/en.md

---
stub: true
tags:
  - april fools
---

# osu!coin

![multiple osu!coins](img/coins2x.png)![a single osu!coin](img/coin2x.png)

> The **osu!coin** was a joke, in-game currency mocking other free-to-play monetization schemes. It was introduced for April Fools, 2015, and allowed players to play a single [beatmap](/wiki/Beatmaps/) in exchange for an osu!coin.

Reviewer: ```suggestion
The **osu!coin** was a ab joke in-game currency mocking other free-to-play monetization schemes. It was introduced for April Fools, 2015, and allowed players to play a single [beatmap](/wiki/Beatmaps) in exchange for an osu!coin.
```

## Release

The [official release statement](https://osu.ppy.sh/home/news/2015-03-31-osucoins) explains that the osu!coin was introduced as a way to fund ![][flag_AU] [peppy](https://osu.ppy.sh/users/2)'s private jet as the current monetization scheme wouldn't allow him to buy it in his lifetime. The statement continues, explaining that each new player would start off with 10 osu!coins, and would be given 10 more for each passing day. Existing users would get an additional two osu!coins as an added bonus. Players could then gain more osu!coins by obtaining combo<!--TODO: link to combo multiplier-->. A single osu!coin would be obtained for every 100x combo.

> When players would run out of osu!coins they would be unable to play any beatmaps. Though they could purchase more—out of their own will—with the exchange rates:

Reviewer: ```suggestion
When players would run out of osu!coins, they would be unable to play any beatmaps. Though they could purchase more—out of their own will—with the exchange rates:
```

- 1USD = 1 osu!coin
- 10USD = 10 osu!coins

> - 999USD = 1000 osu!coins (0.001% savings!)

Reviewer: ```suggestion
- $999 USD = 1000 osu!coins
```
ik the (0.0001% savings!) part was part of the joke, but unless we put it in quotes, it won't work with formal tone.

> - 1USD = 1 osu!coin
> - 10USD = 10 osu!coins
> - 999USD = 1000 osu!coins (0.001% savings!)
> - first unborn child = infinite osu!coins (unborn child will work for osu!)

Reviewer: ```suggestion
- $1 USD = 1 osu!coin
- $10 USD = 10 osu!coins
- $999 USD = 1000 osu!coins (0.001% savings!)
- first unborn child = infinite osu!coins (unborn child will work for osu!)
```

> The following day, peppy rolled back the implementation of osu!coins, opened a new subdomain for the osu!web redesign and ended with [this remark](https://osu.ppy.sh/home/changelog/cuttingedge/20150401).

Reviewer: ```suggestion
The following day, peppy rolled back the implementation of osu!coins, opened a new subdomain for the osu!web redesign and ended with a [remark regarding bug fixes and minor improves](https://osu.ppy.sh/home/changelog/cuttingedge/20150401).
```

Reviewer 2: ```suggestion
The following day, peppy rolled back the implementation of osu!coins, opened a new subdomain for the osu!web redesign and ended with a [remark regarding bug fixes and minor improves](https://osu.ppy.sh/comments/121803).
```

We can also update the link to peppy's comment. This suggestion is on top of paper-toaster's.

## Reception

> Though the release of the osu!coin added additional graphical elements, it had no practical effect on the gameplay experience as players could keep playing past the zero coin limit. Conversely, there was no way to pay for extra osu!coins either (though the option was listed as available).

Reviewer: ```suggestion
Though the release of the osu!coin added additional graphical elements, it had no practical effect on the gameplay experience, as players could keep playing past the zero coin limit. Conversely, there was no way to pay for extra osu!coins either (though the option was listed as available).
```

> The update was generally well-recevied by players as a welcome change of pace from the otherwise strict stream of [news](/news/).

Reviewer: ```suggestion
The update was generally well-recevied by players as a welcome change of pace from the otherwise strict stream of news.
```
link here isn't really needed
